src/core/Trader.ts

```diff
@@ -18,13 +18,13 @@ import { AuthenticatedExchangeAPI } from '../exchanges/AuthenticatedExchangeAPI'
 import { BookBuilder } from '../lib/BookBuilder';
 import { Level3Order, LiveOrder, OrderbookState } from '../lib/Orderbook';
 import { CancelOrderRequestMessage,
-         isStreamMessage,
-         MyOrderPlacedMessage,
-         PlaceOrderMessage,
-         StreamMessage,
-         StreamMessageLike,
-         TradeExecutedMessage,
-         TradeFinalizedMessage } from './Messages';
+    isStreamMessage,
```

Comment:
Unnecessary whitespace changes

Reply:
This has been reverted, not sure why it's still showing in changed files...